k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -5773,6 +5773,39 @@ spec:
                     description: GCPNetworkingSpec is the specification of GCP's native
                       networking mode, using IP aliases.
                     type: object
+                  kindnet:
+                    description: KindnetNetworkingSpec configures Kindnet settings.
```

Comment:
Nit: an annoying thing about go docs mapping to OpenAPI docs is that we probably don't want to follow normal go conventions for comments.  The reader of the OpenAPI docs doesn't see  the name `KindnetNetworkingSpec` on the struct, because OpenAPI doesn't have structs (or we don't use them)